linters/prettier/plugin.yaml

```diff
@@ -28,6 +19,27 @@ lint:
           batch: true
           in_place: true
           formatter: true
+          files:
+            - typescript
+            - yaml
+            - css
+            - sass
+            - html
+            - markdown
+            - json
+            - javascript
+            - graphql
+            - prettier_supported_configs
+        - name: format-xml
+          output: rewrite
+          run: prettier --plugin=@prettier/plugin-xml -w ${target}
+          success_codes: [0]
+          batch: true
+          in_place: true
+          formatter: true
+          version: ">=3.0.0"
```

Comment:
Calling out for posterity that the latest @prettier/plugin-xml only works on prettier >=3.0.0, and you have to manually downgrade in order to support older versions